File: appjava/src/main/java/com/byoutline/kickmaterial/model/ProjectVideo.java
package com.byoutline.kickmaterial.model;

import org.parceler.Parcel;

/**
 * @author Sebastian Kacprzak <sebastian.kacprzak at byoutline.com>
 */
@Parcel(Parcel.Serialization.FIELD)
public class ProjectVideo {
    public int id;
    public String status;
    public String high;
    public String base;
    public String webm;
    public int width;
    public int height;
    public String frame;
}

File: app/src/main/java/com/byoutline/kickmaterial/model/ProjectVideo.kt
package com.byoutline.kickmaterial.model


import paperparcel.PaperParcel
import paperparcel.PaperParcelable

/**
 * @author Sebastian Kacprzak <sebastian.kacprzak at byoutline.com>
 */
@PaperParcel
class ProjectVideo : PaperParcelable {
    var id: Int = 0
    var status: String? = null
    var high: String? = null
    var base: String? = null
    var webm: String? = null
    var width: Int = 0
    var height: Int = 0
    var frame: String? = null

    companion object {
        @JvmField val CREATOR = PaperParcelProjectVideo.CREATOR
    }
}
